cmd/podman/images/rm.go

```diff
@@ -82,6 +82,9 @@ func rm(cmd *cobra.Command, args []string) error {
 			}
 		}
 		registry.SetExitCode(report.ExitCode)
+		if imageOpts.Force && registry.GetExitCode() == 1 {
```

Comment:
This should only happen if the error is image does not exist, and this is the only error returned.